tests/e2e/defaultvmgroup_test.go

```diff
@@ -57,38 +65,106 @@ var _ = Suite(&DefaultVMGroupTestSuite{})
 // Test step:
 // Add a vm to the _DEFAULT Vmgroup
 func (s *DefaultVMGroupTestSuite) TestAddVMToDefaultTenant(c *C) {
-	log.Printf("START: defaultvmgroup.TestAddVMToDefaultTenant")
+	misc.LogTestStart(c.TestName())
 
-	out, err := admincli.AddVMToVMgroup(s.esxIP, con.DefaultVMgroup, s.hostName)
+	out, err := admincli.AddVMToVMgroup(s.config.EsxHost, admincliconst.DefaultVMgroup, s.config.DockerHostNames[1])
 	c.Assert(err, IsNil, Commentf(out))
 	c.Assert(strings.TrimRight(string(out), "\n"), Equals, ErrorMsg, Commentf("Unexpected Behavior: "+
 		"We are able to add vm to the _DEFAULT tenant which is NOT allowed as per current spec."))
 
-	log.Printf("END: defaultvmgroup.TestAddVMToDefaultTenant")
+	misc.LogTestEnd(c.TestName())
 }
 
 // Test step:
 // Remove a vm from the _DEFAULT Vmgroup
 func (s *DefaultVMGroupTestSuite) TestRemoveDefaultTenantVMs(c *C) {
-	log.Printf("START: defaultvmgroup.TestRemoveDefaultTenantVMs")
+	misc.LogTestStart(c.TestName())
 
-	out, err := admincli.RemoveVMFromVMgroup(s.esxIP, con.DefaultVMgroup, s.hostName)
+	out, err := admincli.RemoveVMFromVMgroup(s.config.EsxHost, admincliconst.DefaultVMgroup, s.config.DockerHostNames[1])
 	c.Assert(err, IsNil, Commentf(out))
 	c.Assert(strings.TrimRight(string(out), "\n"), Equals, ErrorMsg, Commentf("Unexpected Behavior: "+
 		"We are able to remove vm from the _DEFAULT tenant which is NOT allowed as per current spec."))
 
-	log.Printf("END: defaultvmgroup.TestRemoveDefaultTenantVMs")
+	misc.LogTestEnd(c.TestName())
 }
 
 // Test step:
 //  Replace a vm from the _DEFAULT Vmgroup
 func (s *DefaultVMGroupTestSuite) TestReplaceDefaultTenantVMs(c *C) {
-	log.Printf("START: defaultvmgroup.TestReplaceDefaultTenantVMs")
+	misc.LogTestStart(c.TestName())
 
-	out, err := admincli.ReplaceVMFromVMgroup(s.esxIP, con.DefaultVMgroup, s.hostName)
+	out, err := admincli.ReplaceVMFromVMgroup(s.config.EsxHost, admincliconst.DefaultVMgroup, s.config.DockerHostNames[1])
```

Comment:
I didn't get you. You are asking why I am using s.config.DockerHostNames[1]? Nothing specific, trying to replace DockerHostNames[1] and not DockerHostNames[1]